styleguide/_themes/derek/scss/globalElements/ceilingLp.scss

```diff
@@ -1,25 +1,33 @@
 .ceilingLp {
   background: $white;
   box-shadow: 0 6px 12px $black-shadow;
-  line-height: 47px;
+
+  .navbar-phoneicon,
+  .navbar-chaticon,
+  .navbar-emailicon {
```

Comment:
I'd just do something like `.navbarLp-phoneicon` instead so we don't have to deal with nesting here.